Introductory Tests/udp_class.py

```diff
@@ -0,0 +1,93 @@
+# General
+import numpy as np
+
+# For computing acceleration and potential
+import polyhedral_gravity as model
+
+# For working with the mesh
+import mesh_utility
+
+# For Plotting
+import pyvista as pv
+
+class udp_obj:
+    
+    def __init__(self,density, r_T, t_end, t_0, dt, lb, ub):
```

Comment:
Could you add some docstrings? You can use this to get a good format

https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring